packages/server/graphql/intranetSchema/intranetSchema.ts

```diff
@@ -92,7 +93,8 @@ const mutation = new GraphQLObjectType<any, GQLContext>({
       stripeUpdateInvoiceItem,
       stripeInvoiceFinalized,
       updateEmail,
-      updateWatchlist
+      updateWatchlist,
+      updateOAuthRefreshTokens
```

Comment:
Maybe a better name is `updateAtlassianOAuthRefreshTokens` in case we need to update GitHub in the future?

Reply:
oh that's a nice catch. TIL: result of `loadMany` has to be filtered [`errorFilter`](https://github.com/ParabolInc/parabol/blob/8dbff9b4c6d19467c96b323be63f61e2926dc0a5/packages/server/graphql/errorFilter.ts)

Reply:
let's keep it as is for now, we can decide whether we want to put github/gitlab logic in it later or break em out into their own